client/views/jobs/job.js

```diff
@@ -2,11 +2,27 @@ Template.job.events({
   'click #job-deactivate': function(event, template) {
     event.preventDefault();
     Modal.show('jobDeactivate', template.data);
+  },
+
+  'click #im-interested': function(event, template) {
+    event.preventDefault();
+
+    Meteor.call("registerJobInterest", template.data._id, function(error, result) {
+      //Modal.hide("jobDeactivate");
+    });
   }
 });
 
 Template.job.helpers({
   'hasLabel': function() {
     return this.jobType || this.remote || this.featured;
+  },
+
+  'notInterested': function () {
```

Comment:
minor quibble - i think 'interestedIn' would be a better name for the helper - 'notInterested' isnt quite logically correct as users are expressing 'i'm interested in this' not 'i'm not interested in this'